NuRadioReco/framework/sim_station.py

```diff
@@ -34,6 +35,14 @@ def get_simulation_weight(self):
     def set_simulation_weight(self, simulation_weight):
         self.__simulation_weight = simulation_weight
 
+    def get_axis_distance(self):
+        if not self.has_parameter(astP.position_vB_vvB):
+            logger.error("sim station position in vB vvB v system is not set")
+            raise ValueError("sim station position in vB vvB v system is not set")
+
+        position_vB_vvB = self.get_parameter(astP.position_vB_vvB)
+        return np.sqrt(position_vB_vvB[0] ** 2 + position_vB_vvB[1] ** 2)
+
```

Comment:
Isn't this deprecated?

Reply:
using the parameter it is deprecated. do you want to keep the functionality itself?

Reply:
I prefer getting rid of it. I don't like the idea of mixing the detector description and the event.

Reply:
okay